cmd/kaf/consume.go

```diff
@@ -40,6 +42,7 @@ func init() {
 	rootCmd.AddCommand(consumeCmd)
 	consumeCmd.Flags().StringVar(&offsetFlag, "offset", "oldest", "Offset to start consuming. Possible values: oldest, newest.")
 	consumeCmd.Flags().BoolVar(&raw, "raw", false, "Print raw output of messages, without key or prettified JSON")
+	consumeCmd.Flags().StringVarP(&filter, "filter", "F", "", "Filter output of messages, prettier than grep")
```

Comment:
tbh i would rather like to see this more specific, e.g. filtering by key, so you can actually search for something.

if it were for actually cases similar to grep, i would rather try to add support for outputting data line-by-line, so grep can be used.